buildout.cfg

```diff
@@ -0,0 +1,59 @@
+### BUILDOUT CONFIG FILE ###
+
+[buildout]
+# DIRECTORIES
+bin-directory = bin
+parts-directory = bin/parts
+develop-eggs-directory = bin/dev-eggs
+
+# PARTS
+parts =
+    tests
+    python
+
+# EGGS
+# packages used project-wide
+eggs =
```

Comment:
Really minor, but there's no way to extract this from `requirements.txt` is there?

Reply:
The answer is no.

Reply:
Yes, unfortunately. I looked for some ways of doing that when I discovered buildout but had to forget the idea...